docs/okeps/okep-5377-extend-udn-to-support-multiple-cluster-subnets-in-layer3-topology.md

```diff
@@ -0,0 +1,284 @@
+# OKEP-5377: Extend UDN to Support Multiple Cluster Subnets in Layer3 Topology
+
+* Issue: [#5377](https://github.com/ovn-org/ovn-kubernetes/issues/5377)
+
+## Problem Statement
+
+UDN currently supports only one cluster subnet per IP family in Layer3 topology. This limits IP
+address planning and scalability for large clusters.
+
+In real-world environments, multiple subnets per IP family are often required for several reasons:
+- **IP exhaustion:** A single CIDR may not be large enough to cover all nodes, especially in large-scale deployments.
+- **Progressive growth:** Clusters may be deployed with limited IP space initially and need to grow
+  incrementally over time without recreating the network.
+- **Pod density variance:** Nodes with different resource capacities may need subnets of different
+  sizes. Supporting multiple subnets enables better IP utilization by allocating appropriately sized
+  subnets per node (based on node hints).
+  _Note: Using different `hostSubnet` sizes is out of scope for this OKEP._
+
+## Goals
```

Comment:
Can you explicitly list support for UDN and CUDN? Can you also explicitly state support for primary and secondary CUDN/UDNs?